firmware/serial_debugger/debug_modules.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+message GPIO {
+  uint32 pin = 1;
+  bool value = 2;
+}
+
+message ATDC {
```

Comment:
Before this was named ADC. Got the following error when adding the following changes to the current PR. 

1. target_link_libraries("BMS_app.elf" "BMS_stm32cube" "BMS_can" "BMS_commit_info" "debug_modules") in firmware/thruna/BMS/CMakeLists.txt
2. #include "debug_modules.pb.h" in firmware/thruna/BMS/src/cubemx/Src/main.c
3. GPIO gpio = GPIO_init_zero; in main(void) of firmware/thruna/BMS/src/cubemx/Src/main.c

![image](https://github.com/UBCFormulaElectric/Consolidated-Firmware/assets/64294628/a8b1c949-8e6d-483c-bd7d-d34042dc978a)

Seems like this is the problem: https://stackoverflow.com/questions/3720034/error-expected-before-token

Changing the name fixed it. Do we care to fix with previous naming?

Reply:
Changed to ADC_Channel